client/src/rpb/granular_view.js

```diff
@@ -67,6 +67,11 @@ class GranularView extends React.Component {
     } = this.props;
 
     const non_dept_key_cols = _.reject(sorted_key_columns, { nick: "dept" });
+    const dropdown_filterable_cols = _.reject(
+      sorted_key_columns,
+      ({ nick }) =>
+        nick === "dept" || nick === "prgm" || nick === "desc" || nick === "tp"
+    );
```

Comment:
This is covering:
Authorities and Expenditures (vote/stat description)
Spending by Program (program)
Transfer Payments (transfer payment name)
Tabled Estimates (vote/stat description)
Full-Time Equivalents (FTEs) by Program (program)
Program Expenditures by Authority Type (program)
Program Expenditures by Standard Object (program)